src/utils/assets.js

```diff
@@ -277,6 +278,16 @@ export const getAssetOption = (
   };
 };
 
+export const getAssetOptionKey = (option: ?AssetOption) => {
+  if (!option) return '';
+  return `${option.chain}-${option.address}`;
```

Comment:
```suggestion
  return `${option.chain}-${option.address ?? option.contractAddress}`;
```

Although the `AssetOption ` type definitely needs to be visited.